packages/atomic/src/components/insight/atomic-insight-interface/atomic-insight-interface.new.stories.tsx

```diff
@@ -0,0 +1,851 @@
+/* eslint-disable @cspell/spellchecker */
+import {parameters} from '@/storybook-utils/common/common-meta-parameters';
+import {renderComponent} from '@/storybook-utils/common/render-component';
+import {wrapInInsightInterface} from '@/storybook-utils/insight/insight-interface-wrapper';
+import type {Meta, StoryObj as Story} from '@storybook/web-components';
+import {html} from 'lit';
+
+const {decorator, play} = wrapInInsightInterface();
+
+const meta: Meta = {
+  component: 'atomic-insight-interface',
+  title: 'Atomic/Insight/InsightInterface',
+  id: 'atomic-insight-interface',
+
+  render: renderComponent,
+  decorators: [decorator],
+  parameters,
+  argTypes: {
+    'attributes-user-id': {
```

Comment:
C'est tu normal que les attributes soit les memes que les user actions? 🤔 
![image](https://github.com/user-attachments/assets/89d4905f-1f6b-4d9d-bb44-89fc980b081c)

J'imagine qu'ont peux pas mettre les attributes des sub-sub-components hein 👯

Reply:
wooops I assumed those also applied to the atomic-insight-interface, but maybe I copied those by mistake 🤦